### DISCUSSION TITLE
Hybrid / Mobile application support via the Ionic / Capacitor Framework

### DISCUSSION THREAD

**Discussion by betterrx-developers** [livewire/livewire#7341] [3. Feature ideas]
Currently Ionic / Capacitor supports converting Angular, React and Vue applications to mobile.  It would be awesome to have same functionality with Livewire / AlpineJS applications.

**Comment by RyanThompson** [comment]
I suppose only AlpineJS could be supported because Livewire is full-stack.

**Comment by ghost** [comment]
We have had some success pairing Livewire with Ionic. However, there are some inconveniences. Ionic is built with Web Components, and due to the nature of Web Components, Livewire is not aware of any attributes (such as classes) applied by Ionic to the HTML elements. A workaround is to use wire:ignore whenever there is reactivity, but this is not ideal.

**Reply by ghost** [reply]
Notably, we are not using Capacitor, but only the UI part of the Ionic framework. The app itself is a Progressive Web App.
